vault/logical_system.go

```diff
@@ -621,6 +622,38 @@ func NewSystemBackend(core *Core, config *logical.BackendConfig) (logical.Backen
 				HelpSynopsis:    strings.TrimSpace(sysHelp["rewrap"][0]),
 				HelpDescription: strings.TrimSpace(sysHelp["rewrap"][1]),
 			},
+
+			&framework.Path{
+				Pattern: "config/audited-headers/(?P<header>.+)",
```

Comment:
Thinking about this, I wonder if we should actually have any public-facing parts of this refer to auditing configuration at a top-level and header configuration at a lower level. So this would become `config/auditing/headers` (or maybe `config/auditing/request-headers` to be explicit). That way if we want to add any other auditing-related configuration at a later time we don't need a totally separate path. (For instance, if we want to configure the system default for HMAC-ing accessors.)